pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -603,6 +623,14 @@ func getSelectorsFromPodInfo(pod *corev1.Pod, status *corev1.ContainerStatus) []
 		makeSelector("pod-name:%s", pod.Name),
 		makeSelector("container-name:%s", status.Name),
 		makeSelector("container-image:%s", status.Image),
+		makeSelector("pod-container-count:%s", podContainerCount),
+		makeSelector("pod-init-container-count:%s", podInitContainerCount),
```

Comment:
nit: getPodImages doesn't need to return the count since we can just use the length of the returned images
```suggestion
		makeSelector("pod-container-count:%s", len(podImages)),
		makeSelector("pod-init-container-count:%s", len(podInitImages)),
```